driver_lower:
  ros__parameters:
    pro_motors: [31, 40, 41, 42, 43] #Motor ids that are the dynamixel pro motors
    roboclaw_motors: [60, 61, 62, 63] #Note: These are arbitrary (not used by actual motors/boards), but need to remain consistent
    dxl_usb_id: "/dev/ttyUSB0" #USB Device containing dynamixel motors
    rc_usb_ids: ["/dev/ttyUSB0", "/dev/ttyUSB1"] # USB devices containing roboclaw motors
    usb_baudrate: 1000000 #Baudrate, probably wont be changed